Something/WebApplication/wwwroot/js/apps/authentication/Authentication.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from "react";
+
+import { AuthenticationForm } from "./components/AuthenticationForm/AuthenticationForm";
+import { Splash } from "./components/Splash/Splash";
+
+import "./../Global.scss";
```

Comment:
Global.scss should be included in each of you scss files if they are shared variables.